ReleaseNotes.md

```diff
@@ -8,6 +8,36 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 05/16/2023 -
+##### Version microsoft/oms:3.1.8 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.8 (linux)
+##### Version microsoft/oms:win-3.1.8 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.8 (windows)
+##### Current dependencies
+- Linux
+  - CBL-Mariner 2.0
+  - Golang - 1.20.3
+  - Ruby - 3.1.3
+  - MDSD - 1.26.1-build.master.97
+  - Telegraf - 1.26.0
+  - Fluent-bit - 2.0.9
+  - Fluentd - 1.14.6
+- Windows
+  - Ruby - 3.1.1
+  - Fluent-bit - 2.0.5
+  - Telegraf - 1.24.2
+  - Fluentd - 1.14.6
+  - Windows AMA - 46.3.2
+  - Golang - 1.20.3
+##### Code change log
+- Common
+  - Add KataRuntimePod check in kubernetes_container_inventory
```

Comment:
Kata runtime pod is specific to Linux. Lets move this under linux

Reply:
moved